src/plots/geo/layout_defaults.js

```diff
@@ -162,14 +164,15 @@ function handleGeoDefaults(geoLayoutIn, geoLayoutOut, coerce, opts) {
         // Only works for:
         //   USA states at 110m
         //   USA states + Canada provinces at 50m
-        coerce('showsubunits', true);
+        //   !!
```

Comment:
Is this a comment?

Reply:
https://github.com/plotly/plotly.js/pull/4419/commits/b5b9484bc6760faf5b4a93ab7067927559632ae5